pkg/cloud/instance_test.go

```diff
@@ -143,23 +143,23 @@ var _ = Describe("Instance", func() {
 
 		It("returns errors occurring while fetching service offering information", func() {
 			expectVMNotFound()
-			sos.EXPECT().GetServiceOfferingID(dummies.CSMachine1.Spec.Offering.Name).Return("", -1, unknownError)
+			sos.EXPECT().GetServiceOfferingID(dummies.CSMachine1.Spec.Offering.Name, gomock.Any()).Return("", -1, unknownError)
```

Comment:
```suggestion
			sos.EXPECT().GetServiceOfferingID(dummies.CSMachine1.Spec.Offering.Name, cloudstack.WithZone(dummies.XXX.zoneID)).Return("", -1, unknownError)
```
Why not expecting a real value instead of Any()?

Reply:
We don't care the value here, so we use gomock.Any to refer it. 

If we use `cloudstack.WithZone(...`, we will get the follwowing error: 
`
 Got: [0x229b1c0] ([]interface {})
  Want: is equal to 0x229b1c0 (cloudstack.OptionFunc)
`

I am sure there is a work-around on above error, since we are using that value, I used gomock.Any() to refer it.